tests/base.py

```diff
@@ -98,6 +99,11 @@ def setUp(self):
             resource = Resource(name='Markers',
                                 quantity=3)
             resource.save()
+            resource_2 = Resource(
+                name="Pencils",
+                quantity=5
+            )
+            resource_2.save()
```

Comment:
@verenceLola is there any reason why you saved another resource to the DB instead of using the one above it?

Reply:
Thanks. I got the explanation during the sync.